### ISSUE TITLE
Bulk New Term Requests and New Synonym Requests

### ISSUE THREAD

**Issue by evanc1648** [obophenotype/human-phenotype-ontology#11643]
Hello, I've attached two spreadsheets (one for NTRs (61) and one for synonyms (7)) since there are a large number of term requests. These term requests are part of the project for the custom phenotypic profile for Fanconi anemia (Pre-print: https://www.medrxiv.org/content/10.64898/2026.02.10.26346018v1). 

[HPO_NTR_Requests_upload.xlsx](https://github.com/user-attachments/files/29394208/HPO_NTR_Requests_upload.xlsx)
[HPO_Synonym_Requests_upload.xlsx](https://github.com/user-attachments/files/29394207/HPO_Synonym_Requests_upload.xlsx)

**Comment by mellybelly** [comment]
@pnrobinson any comments or questions? the manuscript needs these ;-). perhaps @matentzn can help?

**Comment by matentzn** [comment]
I can do these, but @evanc1648 can you please create Google sheets, share them with me, and we will go back and forth to bring them into the right shape before I will pass them to Peter for review?

**Comment by pnrobinson** [comment]
@matentzn let me do the NTRs, could you do the synonyms. @mellybelly recovering from [Shai-Hulud](https://about.gitlab.com/blog/shai-hulud-copycat-campaign-targets-python-developers/), could not do much the last few weeks...

**Comment by matentzn** [comment]
@pnrobinson there you go: https://github.com/obophenotype/human-phenotype-ontology/pull/11652

**Comment by pnrobinson** [comment]
@mellybelly @evanc1648 
I am going to make separate issues for some items that need some discussion.
I created a label "Fanconi" that you can use to search, e.g. 
https://github.com/obophenotype/human-phenotype-ontology/issues/11653
Let's try to finish this before the next HPO release in 3 weeks if that will match with your schedule (send separate email to discuss time pressure if any)

**Comment by pnrobinson** [comment]
@evanc1648 
adding

HPO ID	HP:6001488
Label	Abnormal vulvar morphology
Definition	Any structural abnormality of the vulva, the external female genitalia comprising the mons pubis, labia, clitoris, and vaginal vestibule.
Comment	
Parent term(s):	Abnormal female external genitalia morphology (HP:0000055)
PMIDs	
Synonyms	Abnormal vulva morphology(NONE);Vulvar malformation(NONE);Abnormality of the vulva(NONE)
ORCID	0000-0002-0736-9199

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001493
- label: Cranial exostosis
- parents: [Exostoses (HP:0100777)](https://hpo.jax.org/app/browse/term/HP:0100777)
- pmid: PMID:20043756|PMID:28614150
- def: A benign bony outgrowth projecting from the outer surface of one or more of the cranial bones.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Skull exostosis(NONE)|Cranial exostoses(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001494
- label: Cubital dimple
- parents: [Skin dimple (HP:0010781)](https://hpo.jax.org/app/browse/term/HP:0010781)
- pmid: 
- def: A cutaneous indentation (dimple) located in the cubital fossa, which is a triangular depression located on the anterior aspect of the elbow.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Antecubital dimple(NONE)|Cubital pit(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Decreased circulating sex hormone concentration => Probably best to take one of the specific "decreased" terms from this subhierarchy?
[Abnormal circulating sex hormone concentration HP:0033799](https://hpo.jax.org/browse/term/HP:0033799)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001495
- label: Distal interphalangeal crease effacement of the 5th finger
- parents: [Absent distal interphalangeal creases (HP:0001032)](https://hpo.jax.org/app/browse/term/HP:0001032)
- pmid: 
- def: Reduced visibility or partial loss of the distal interphalangeal flexion crease of the fifth finger, without complete absence.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Faded 5th finger DIP crease(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001496
- label: Duodenal web
- parents: [Abnormal duodenum morphology (HP:0002246)](https://hpo.jax.org/app/browse/term/HP:0002246)
- pmid: PMID:33370983|PMID:31839373|PMID:21478597
- def: A congenital partial obstruction of the duodenal lumen by a membrane or obstructing septum (web) formed from mucosa and submucosa.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Duodenal membrane(NONE)|Duodenal diaphragm(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001497
- label: Eccentric nipple
- parents: [Abnormal nipple morphology (HP:0004404)](https://hpo.jax.org/app/browse/term/HP:0004404)
- pmid: 
- def: A morphological abnormality in which the nipple is positioned off-center relative to the areola.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Off-center nipple(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001499
- label: Eyebrow asymmetry
- parents: [Abnormal eyebrow morphology (HP:0000534)](https://hpo.jax.org/app/browse/term/HP:0000534)
- pmid: 
- def: A difference in shape, size, or position between the left and right eyebrows.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Asymmetric eyebrows(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001500
- label: Labor dystocia
- parents: [Abnormal delivery (HP:0001787)](https://hpo.jax.org/app/browse/term/HP:0001787)
- pmid: PMID:33448772|PMID:20633305
- def: Abnormally slow progression or arrest of the normal stages of labor.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Failure to progress in labor(NONE)|Protracted labor(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001501
- label: Foregut duplication cyst
- parents: [Gastrointestinal duplication (HP:0011140)](https://hpo.jax.org/app/browse/term/HP:0011140)
- pmid: PMID:28642639|PMID:29390328
- def: A congenital cystic malformation derived from the embryonic foregut and attached to or arising from the upper gastrointestinal tract (esophagus, stomach, or proximal duodenum).
- comment: Foregut duplication cysts (FDCs) are rare congenital anomalies of primitive foregut origin that arise during early embryonic development. FDCs comprise approximately 10% of all mediastinal tumors and can occur at any level of the alimentary track. Computed tomography (CT) is widely used as a modality for the evaluation and diagnosis of duplication cysts.
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Foregut duplication(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001502
- label: Hyperpigmented palmar creases
- parents: [Abnormality of the palmar creases (HP:0010490)](https://hpo.jax.org/app/browse/term/HP:0010490)
- pmid: PMID:33533144
- def: Increased pigmentation localized to the flexion creases of the palm.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Dark palmar creases(NONE)|Hyperpigmentation of palmar creases(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001503
- label: Hypoplastic clivus
- parents: [Abnormality of the clivus (HP:0010558)](https://hpo.jax.org/app/browse/term/HP:0010558)
- pmid: PMID:29622552
- def: Congenital underdevelopment of the clivus, the bony surface of the basisphenoid and basiocciput that forms part of the skull base.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Clivus hypoplasia(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001504
- label: Increased mammary adipose tissue
- parents: [Increased adipose tissue (HP:0009126)](https://hpo.jax.org/app/browse/term/HP:0009126) | [Abnormal breast morphology (HP:0031093)](https://hpo.jax.org/app/browse/term/HP:0031093)
- pmid: 
- def: An abnormal increase in adipose tissue specifically localized to the mammary region.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Mammary fat excess(NONE)|Increased breast adipose tissue(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001505
- label: Increased palmar soft tissue
- parents: [Abnormal palm morphology (HP:0100871)](https://hpo.jax.org/app/browse/term/HP:0100871)
- pmid: 
- def: Increased soft tissue bulk of the palm, giving a fleshy or full appearance.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Fleshy palm(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001506
- label: Interphalangeal crease effacement of the thumb
- parents: [Abnormal finger flexion crease (HP:0006143)](https://hpo.jax.org/app/browse/term/HP:0006143)
- pmid: 
- def: Reduced visibility or partial loss of the interphalangeal flexion crease of the thumb, without complete absence.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Effacement of the interphalangeal crease of the thumb(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001507
- label: Irregular anterior hairline
- parents: [Abnormality of the frontal hairline (HP:0000599)](https://hpo.jax.org/app/browse/term/HP:0000599)
- pmid: 
- def: An anomaly of the frontal hairline in which the border between hair-bearing and non-hair-bearing forehead skin follows an uneven, jagged contour rather than a smooth curve.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Uneven frontal hairline(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001508
- label: Lateral deviation of the uvula
- parents: [Abnormal uvula morphology (HP:0000172)](https://hpo.jax.org/app/browse/term/HP:0000172)
- pmid: 
- def: An abnormal position of the uvula, which normall is located in the midline, to either the left or the right.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Deviated uvula(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001509
- label: Limited metacarpophalangeal flexion of 2nd finger
- parents: [Decreased finger mobility (HP:0006135)](https://hpo.jax.org/app/browse/term/HP:0006135)
- pmid: 
- def: Reduced ability to flex the metacarpophalangeal joint of the second finger.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms:

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001510
- label: Limited metacarpophalangeal flexion of 3rd finger
- parents: [Decreased finger mobility (HP:0006135)](https://hpo.jax.org/app/browse/term/HP:0006135)
- pmid: 
- def: Reduced ability to flex the metacarpophalangeal joint of the third finger.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms:

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001511
- label: Limited metacarpophalangeal flexion of 4th finger
- parents: [Decreased finger mobility (HP:0006135)](https://hpo.jax.org/app/browse/term/HP:0006135)
- pmid: 
- def: Reduced ability to flex the metacarpophalangeal joint of the fourth finger.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms:

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001512
- label: Limited metacarpophalangeal flexion of 5th finger
- parents: [Decreased finger mobility (HP:0006135)](https://hpo.jax.org/app/browse/term/HP:0006135)
- pmid: 
- def: Reduced ability to flex the metacarpophalangeal joint of the fifth finger.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms:

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001513
- label: Short manubrium of malleus
- parents: [Abnormal malleus morphology (HP:0011454)](https://hpo.jax.org/app/browse/term/HP:0011454)
- pmid: PMID:27428025
- def: Reduced length of the manubrium (handle) of the malleus, the most lateral of the auditory ossicles.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Hypoplastic malleus manubrium(NONE)|Short malleus handle(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 
Adding new term: 
- ID: HP:6001514
- label: Supernumerary thumb flexion crease
- parents: [Supernumerary finger flexion crease (HP:6000109)](https://hpo.jax.org/app/browse/term/HP:6000109)
- pmid: 
- def: The presence of one or more additional flexion creases on the thumb beyond the typical number.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: 
- synonyms: Extra thumb flexion crease(NONE)

**Comment by pnrobinson** [comment]
@evanc1648 @mellybelly 
Closing this -- the above terms were added. I made individual issues labeled with Fanconi for terms I am working on or about which I had questions.
